adapters/adxcg/adxcg.go

```diff
@@ -0,0 +1,130 @@
+package adxcg
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v14/openrtb2"
+	//"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// Builder builds a new instance of the Adxcg adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	bidder := &AdxcgAdapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+// MediaOptiAdapter implements the Bidder interface.
+type AdxcgAdapter struct {
```

Comment:
We prefer the name `adapter`. No comment is necessary.

Reply:
done